CHANGELOG.md

```diff
@@ -3,6 +3,44 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v2.0.0 - [2025/10/14]
+
+### `Added`
+
+- [#124](https://github.com/nf-core/proteinfamilies/pull/124)
+  - Added new subworkflow `MERGE_FAMILIES` that can optionally merge similar (but not redundant) generated protein families. (by @vagkaratzas)
+  - Added new functionality to local module `IDENTIFY_REDUNDANT_IDS` which now also identifies similar family ids, along with their pairwise similarity scores. (by @vagkaratzas)
```

Comment:
I'm a little puzzled by the "family ids". I would expect you to identify "similar families" (and with that, their ids) but can't say since this is the second file I'm looking at.